kotlin/viewregistry/src/main/java/com/squareup/viewregistry/IsModalContainerScreen.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.viewregistry
+
+interface IsModalContainerScreen<out B : Any, out M : Any> {
```

Comment:
Nit: `Is` as a class prefix is strange- it's unclear what it signifies or how it's different from `ModalContainerScreen` (and there's no doc to clarify).

Reply:
It's long been my habit to use the `Is` prefix for characteristic interfaces. Here both `AlertContainerScreen` and `PanelContainerScreen` have the trait that they show modal screens over a base screen.

I need a common api for `AbstractModalContainer` to work with, but so far ViewRegistry can only bind to concrete classes. That is, I can't define separate bindings for `ModalContainerScreen<AlertScreen, *>` and `ModalContainerScreen<BackStackScreen<*>, *>`.

Anyway, yes, kdoc needed.